auth0_flutter_platform_interface/lib/src/web-auth/web_auth_login_options.dart

```diff
@@ -21,20 +25,37 @@ class WebAuthLoginOptions implements RequestOptions {
       this.invitationUrl,
       this.scheme,
       this.useEphemeralSession = false,
-      this.parameters = const {}});
+      this.parameters = const {},
+      this.safariViewController});
 
   @override
-  Map<String, dynamic> toMap() => {
-        'leeway': idTokenValidationConfig?.leeway,
-        'issuer': idTokenValidationConfig?.issuer,
-        'maxAge': idTokenValidationConfig?.maxAge,
-        'audience': audience,
-        'scopes': scopes.toList(),
-        'redirectUrl': redirectUrl,
-        'organizationId': organizationId,
-        'invitationUrl': invitationUrl,
-        'useEphemeralSession': useEphemeralSession,
-        'parameters': parameters,
-        'scheme': scheme
-      };
+  Map<String, dynamic> toMap() {
+    final map = {
+      'leeway': idTokenValidationConfig?.leeway,
+      'issuer': idTokenValidationConfig?.issuer,
+      'maxAge': idTokenValidationConfig?.maxAge,
+      'audience': audience,
+      'scopes': scopes.toList(),
+      'redirectUrl': redirectUrl,
+      'organizationId': organizationId,
+      'invitationUrl': invitationUrl,
+      'useEphemeralSession': useEphemeralSession,
+      'parameters': parameters,
+      'scheme': scheme,
+      ...safariViewController != null
+          ? {
+              'safariViewController': {
+                ...safariViewController?.presentationStyle != null
+                    ? {
+                        'presentationStyle':
+                            safariViewController?.presentationStyle?.value
+                      }
+                    : <String, dynamic>{}
+              }
+            }
+          : <String, dynamic>{}
```

Comment:
This should be implemented as part of the Dart `SafariViewController` class, e.g. as a `toMap()` method, that should simply be called here.